source/about/faq.rst

```diff
@@ -110,9 +111,9 @@ Have a look at our plugin pages to get a quick-start: :doc:`/plugin/index`
 What can't I do with Sponge? / Limitations of Sponge?
 -----------------------------------------------------
 
-Sponge can't be used to create new blocks, textures, mobs on the clientside or any other content which would need
-clientside modifications. SpongeAPI wont support sending mods or plugins to the client for now due to security
-concerns. However you can make use of the ForgeAPI for clients and create Sponge plugins for the serverside.
+Sponge can't be used to create new blocks, textures, mobs on the client side or any other content which would need
+client side modifications. SpongeAPI won't support sending mods or plugins to the client for now due to security
+concerns. However you can make use of the ForgeAPI for clients and create Sponge plugins for the server side.
```

Comment:
Alternatively, we could use the concatenated versions, i.e. client-side and server-side. It may be clearer, what do you think? Also, comma needed after "However".